tools/meta/src/getPackageManager.ts

```diff
@@ -1,11 +1,19 @@
 import fse from 'fs-extra';
+
+export type SupportedPackageManager = `npm` | `yarn` | `pnpm`;
+
 /** Reads the lock file to determine which package manager to use */
-export function getPackageManager(appPath: string): `npm` | `yarn` {
+export function getPackageManager(
+  appPath: string,
+  defaultPackageManager: SupportedPackageManager = 'npm',
```

Comment:
why is defaultPackageManager an arg?

Reply:
Doesn't need to be, but I wanted to add the flexibility of preferring some other default if there's no lock file found